<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeAveragesForm</class>
 <widget class="QWidget" name="TimeAveragesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>447</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Time averages</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QTreeView" name="treeViewAverage">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDelete">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>393</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBoxTimeAverage">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" rowspan="2">
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>13</width>
               <height>128</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1" colspan="3">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <spacer>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>119</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1">
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="labelAverage">
                 <property name="text">
                  <string>Label of time average </string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEditAverage">
                 <property name="toolTip">
                  <string>To give a name for the choosen time average</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="labelStart_3">
                 <property name="toolTip">
                  <string>Code_Saturne key word: NTDMOM(IMOM)</string>
                 </property>
                 <property name="text">
                  <string>Starting type</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="comboBoxStartType"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="labelStart">
                 <property name="toolTip">
                  <string>Code_Saturne key word: NTDMOM(IMOM)</string>
                 </property>
                 <property name="text">
                  <string>Start iteration number
for time average calculation</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="lineEditStart">
                 <property name="toolTip">
                  <string>Code_Saturne key word: NTDMOM(IMOM)</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="labelStartTime">
                 <property name="toolTip">
                  <string>Code_Saturne key word: TTDMOM(IMOM)</string>
                 </property>
                 <property name="text">
                  <string>Start time for
time average calculation</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="lineEditStartTime">
                 <property name="toolTip">
                  <string>Code_Saturne key word: TTDMOM(IMOM)</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0" rowspan="2">
                <widget class="QLabel" name="labelRestart">
                 <property name="toolTip">
                  <string>Code_Saturne key word: IMOOLD(IMOM)</string>
                 </property>
                 <property name="text">
                  <string>If restart, number in preceding calculation
of the time average to use to initialize
the current selected time average.
</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QComboBox" name="comboBoxIMOOLD"/>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="lineEditRestart">
                 <property name="toolTip">
                  <string>Code_Saturne key word: IMOOLD(IMOM)</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="2">
              <spacer>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>119</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="0" column="4" rowspan="2">
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>13</width>
               <height>138</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QListView" name="listViewDrag">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <layout class="QVBoxLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButtonAddVar">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources_pages.qrc">
                 <normaloff>:/new/prefix1/icons/32x32/go-next.png</normaloff>:/new/prefix1/icons/32x32/go-next.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonSuppressVar">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources_pages.qrc">
                 <normaloff>:/new/prefix1/icons/32x32/go-previous.png</normaloff>:/new/prefix1/icons/32x32/go-previous.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="3">
            <widget class="QListView" name="listViewDrop">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources_pages.qrc"/>
 </resources>
 <connections/>
</ui>
